### PR TITLE
Removed code lint warnings/errors :rocket:

### LINKED ISSUE #15: Fix Lint errors and warning in index.html
`index.html` throws many warnings.
Also, the code is bit messy.

### Minimal Inspector Output:

image

### PR DESCRIPTION
**Pull request by 0x48piraj**
## Fixes #15

* Removed warnings (eg. `Warning: <meta> lacks "content" attribute; Warning: <script> inserting "type" attribute`)
* Added `<title>` to surpress the errors. *(Used inspector and read rules)*
* Linted in a self explanatory fashion. eg. `</script>` and `<!--...>`
* Used proper spacing to beautify the code also. :heart:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
-    <script src="js/level.js"></script>
-    <!--<meta http-equiv="refresh" content="5">-->
-    <link rel="stylesheet" href="css/tiles.css">
-    <title></title>
-  </head>
-  <body>
-    <div id="tiles">
-    </div>
-  </body>
+<head>
```
Comment (xeon-zolt) [inline index.html]: An HTML element's start tag may be omitted if the first thing inside the HTML element is not a comment.

sometimes omitting elements may lead to different renders on different browsers

Comment (niccokunzmann) [reply index.html]: Please add `<html>`

Comment (0x48piraj) [reply index.html]: I have <html> in the start. Please can you specify where to add ?

**Comment by niccokunzmann** [comment]
If someone else likes to add the `<html>` tag at the start, we will merge it.

**Pull request merged**